app/internal/comment.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+import json
+from typing import Dict, Tuple
+
+from sqlalchemy.orm.session import Session
+
+from app.database.models import Comment, Event
+from app.internal import utils
+
+
+def create_comment(session: Session, event: Event, content: str) -> None:
+    """Creates a comment instance in the DB.
+
+    Args:
+        session (Session): DB session.
+        event (Event): Instance of the event to create a comment for.
+        content (str): The content of the comment.
+    """
+    data = {
+        'user': utils.get_current_user(session),
+        'event': event,
+        'content': content,
+        'time': datetime.datetime.now()
+    }
+    utils.create_model(session, Comment, **data)
+
+
+def parse_comment(session: Session,
+                  comment: Comment) -> Tuple[int, Dict[str, str]]:
+    """Returns a dictionary with the comment's info.
+
+    Args:
+        session (Session): DB session.
+        comment (Comment): Comment instance to parse.
+
+    Returns:
+        dict(str: str): Comment's info.
+                        'id' - The comment's.
+                        'avatar' - Commentor's profile image.
+                        'username' - Commentor's username.
+                        'time' - Comment's posting time.
+                        'content' - Comment's content
+    """
+    user = utils.get_user(session, comment.user_id)
+    return comment.id, {
```

Comment:
I suggest that `comment.id` would be part of the dict :) (shallow structures are almost always better)